src/chapters/forms/form-submission-post.rst

```diff
@@ -0,0 +1,98 @@
+POST Form Submission
+====================
+
+Form Submission Using POST
+--------------------------
+Instead of using GET and query parameters to submit form data, we can use POST.
+To submit a form using a POST request, set the form's ``method`` attribute to ``"POST"``.
+Form data submitted via POST will be submitted in the body of the HTTP request.
+Data submitted by GET requests is less secure than POST because GET request URLs
+and the query parameters are cached and logged, possibly leaking sensitive data.
+
+.. admonition:: Example
+
+   Form with ``method="POST"``
+
+   .. sourcecode:: html
+
+      <form action="" method="POST">
+         <label>Username <input type="text" name="username"></label>
+         <label>Team Name <input type="text" name="team"></label>
+         <button>Submit</button>
+      </form>
+
+Send Form Submission to a Server
+--------------------------------
+
+The ``action`` and ``method`` attributes allow us to choose where the form request will be
+sent and what type of request will be sent. How do we configure what happens in response to
+a form submission?
+
+.. index:: ! form handler
+
+**Form handlers** are web server actions that receive, inspect, and process requests.
+They then send a response to the client. For this unit we are going to use form handlers that have already
+been created for us.
+
+.. admonition:: Example
+
+   When submitted, this form will send a POST request to the form handler defined by the
+   ``action`` attribute.
+
+   .. replit:: html
+      :slug: form-post
+      :linenos:
+
+      <form action="https://handlers.education.launchcode.org/request-parrot" method="POST">
+         <label>Username <input type="text" name="username"></label>
+         <label>Team Name <input type="text" name="team"></label>
+         <button>Submit</button>
+      </form>
+
+.. admonition:: Try It!
+
+   1. Open `example form that uses POST <https://form-post--launchcode.repl.co/>`_ in a browser.
+   2. Open the network tab of the developer tools
```

Comment:
This only works in Firefox, mine looks completely different in chrome and safari, it is preserve log that you click also.(line 56) in both chrome and safari.  Not sure how to handle the different browsers, where they all suppose to have Firefox?